### PR TITLE
학사일정 API 구현

### PR DESCRIPTION
**Pull request by sukso96100**
- 단위테스트 포함.
- 문서 갱신
- master 와 동기화

### CODE REVIEW

**Comment by sukso96100** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           

Complexity increasing per file
==============================
- schedules.go  4
- schedules_test.go  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/s-owl/skhuinfo-api/pullRequest?prid=6091219&bid=19944403)

File: schedules_test.go [hunk 1]
```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var SchedulesHttpMock HttpClient = &HttpMock{
```
Comment (sukso96100) [inline schedules_test.go:9]: Codacy found an issue: [var SchedulesHttpMock should be SchedulesHTTPMock](https://app.codacy.com/gh/s-owl/skhuinfo-api/pullRequest?prid=6091219)

File: consts.go [hunk 2]
```diff
@@ -14,4 +14,6 @@ const (
 	MEAL_LIST = SKHU_URL + "uni_zelkova/uni_zelkova_4_3_list.aspx"
 	// 학식 게시판 주소
 	MEAL_BOARD = SKHU_URL + "uni_zelkova/uni_zelkova_4_3_view.aspx?idx="
+	// 학사일정 주소
+	SCHEDULES_URL = SKHU_URL + "calendar/calendar_list_1.aspx"
```
Comment (sukso96100) [inline consts.go:18]: Codacy found an issue: [don't use ALL_CAPS in Go names; use CamelCase](https://app.codacy.com/gh/s-owl/skhuinfo-api/pullRequest?prid=6091219)

File: consts.go [hunk 3]
```diff
@@ -14,4 +14,6 @@ const (
 	MEAL_LIST = SKHU_URL + "uni_zelkova/uni_zelkova_4_3_list.aspx"
 	// 학식 게시판 주소
 	MEAL_BOARD = SKHU_URL + "uni_zelkova/uni_zelkova_4_3_view.aspx?idx="
+	// 학사일정 주소
```
Comment (sukso96100) [inline consts.go:17]: Codacy found an issue: [comment on exported const SCHEDULES_URL should be of the form "SCHEDULES_URL ..."](https://app.codacy.com/gh/s-owl/skhuinfo-api/pullRequest?prid=6091219)

**Comment by sukso96100** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           

Complexity increasing per file
==============================
- schedules_test.go  3
- schedules.go  4
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/s-owl/skhuinfo-api/pullRequest?prid=6091219&bid=19944403)

File: schedules_test.go (hunk 1 above)
Comment (sukso96100) [inline schedules_test.go:9]: Codacy found an issue: [var SchedulesHttpMock should be SchedulesHTTPMock](https://app.codacy.com/gh/s-owl/skhuinfo-api/pullRequest?prid=6091219)

File: consts.go (hunk 2 above)
Comment (sukso96100) [inline consts.go:18]: Codacy found an issue: [don't use ALL_CAPS in Go names; use CamelCase](https://app.codacy.com/gh/s-owl/skhuinfo-api/pullRequest?prid=6091219)

File: consts.go (hunk 3 above)
Comment (sukso96100) [inline consts.go:17]: Codacy found an issue: [comment on exported const SCHEDULES_URL should be of the form "SCHEDULES_URL ..."](https://app.codacy.com/gh/s-owl/skhuinfo-api/pullRequest?prid=6091219)

**Comment by sukso96100** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- schedules.go  4
- schedules_test.go  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/s-owl/skhuinfo-api/pullRequest?prid=6091219&bid=19944403)

**Comment by sukso96100** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- schedules_test.go  3
- schedules.go  4
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/s-owl/skhuinfo-api/pullRequest?prid=6091219&bid=19944403)

**Comment by sukso96100** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- schedules_test.go  3
- schedules.go  4
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/s-owl/skhuinfo-api/pullRequest?prid=6091219&bid=19944403)

**Review by tpdns90321 — APPROVED** [review]

**Pull request merged**